webpack.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
         filename: "ripe-sdk-components-react.min.js?[fullhash]",
         library: "RipeSdkComponentsReact",
         libraryTarget: "umd",
-        publicPath: "/"
+        publicPath: "/dist/"

```

Comment:
Are you sure about this change? We're not using this value (`/dist/`) in any other project.
To be honest I'm confused about the purpose of the `publicPath` key and the docs (https://webpack.js.org/guides/public-path/) do little in helping me understand the purpose of it.
@BeeMargarida Do you have any idea of the use cases of the `publicPath`?

Reply:
Well, I can't seem to replicate the error had in my home PC here in the Mac, so I'll close this PR, since its working fine